lib/index.js

```diff
@@ -12,3 +12,6 @@ export { bmultisigClient } from './clients';
 export { default as chain } from './chain';
 export { default as helpers } from './helpers';
 export { default as plugins } from './plugins';
+export { TxnManager, TxnManagerOptions } from './txnManager';
```

Comment:
This was going to be changed to `TxManager` right? (no `n`)

Reply:
This has been addressed